client/webpack.config.node.js

```diff
@@ -69,7 +58,7 @@ function getExternals() {
 				// Force all monorepo packages to be bundled. We can guarantee that they are safe
 				// to bundle, and can avoid shipping the entire contents of the `packages/` folder
 				// (there are symlinks into `packages/` from the `node_modules` folder)
-				...getMonorepoPackages(),
+				...packagesInMonorepo().map( ( pgk ) => new RegExp( `^${ pgk.name }(/|$)` ) ),
```

Comment:
Nit, but I'm assuming you meant `pkg` instead of `pgk`.

```suggestion
				...packagesInMonorepo().map( ( pkg ) => new RegExp( `^${ pkg.name }(/|$)` ) ),
```